utils/sccli.sh

```diff
@@ -68,7 +68,23 @@ function usage()
 
 function openshift()
 {
+    # Default OpenShift Image and Tag
+    dockerRegistry=${DOCKER_REGISTRY:-docker.io}
+    imageName=${IMAGE_NAME:-openshift/origin}
+    imageTag=${IMAGE_TAG:-v1.1.1}
+
     clean_setup
+
+    # Setup the systemd environment - desired image may have changed
+    sudo sed -i.back "/^IMAGE=*/cIMAGE=\"${imageName}\:${imageTag}\"" /etc/sysconfig/openshift_option
```

Comment:
We need to put a condition here and only run sed if it is required as users can run `sccli openshift` multipletime